packages/terra-form/src/components/ChoiceField.jsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable jsx-a11y/label-has-for */
+
+import React, { PropTypes } from 'react';
+
+import Control from './Control';
+import Field from './Field';
+
+const propTypes = {
+  /**
+   * Choices to populate radio buttosn for
+   */
+  choices: PropTypes.array,
```

Comment:
> The React form libraries I've worked with use children and I think it makes it a lot easier for the consumer

I think this depends on if you are dealing with static choices that are coded or dynamic choices populated from the server.

These options aren't mutually exclusive. Technically what you're describing can already be done use the current design.. ChoiceField is just a higher order component that does that for you. It prevents consumers from having to do the boilerplate code of listing our the radio fields or looping over them and creating them.